### PR TITLE
Feature/cache headers

### PR DESCRIPTION
**Pull request by LindsayYoung**
Adding max age, deployed it to dev but it doesn't add the header.

also saw the hmac header script snuck back in so got rid of that

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -64,7 +48,7 @@ http {
         proxy_set_header X-Forwarded-Host $host;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Script-Name "<%= path %>";
-
+        proxy_set_header Cache-Control "s-maxage=(600)";
```
Comment (ccostino) [inline nginx.conf]: Hmm, so this isn't coming through?

It appears that there might need to be a bit more set based on some examples I'm seeing.

I also came across the [NGINX Caching Guide](https://www.nginx.com/blog/nginx-caching-guide/), which might be useful in figuring out what to set (it also links to more specific documentation as well).

It looks like there might be a bit more that needs to be configured?  [This DigitalOcean guide](https://www.digitalocean.com/community/tutorials/understanding-nginx-http-proxying-load-balancing-buffering-and-caching#configuring-proxy-caching-to-decrease-response-times) also has some info around proxy cache configuration.

File: nginx.conf [hunk 2]
```diff
@@ -38,23 +39,6 @@ http {
...
-            value
-          }
-      end
-      def string_to_sign(headers)
-        keys = ["$request_method"]
-        keys.concat(headers.map { |header| "$http_#{header}" })
-        keys.concat(["$request_uri", "$request_body"])
-        return keys.join("\\n")
-      end
-    %>
```
Comment (LindsayYoung) [inline nginx.conf]: Yea, we shouldn't be using this and the other variables come from the manifest.

**Comment by LindsayYoung** [comment]
@jmcarp thank you so much- I was curling the wrong url. The response seem really obvious now but I was just reading the headers. I feel like an idiot but thank you for your help and kindness! Sorry to bother you @cm while I lost my mind

File: nginx.conf [hunk 3]
```diff
@@ -38,33 +38,16 @@ http {
...
 
     <% proxies.each do |path, route| %>
       location <%= path %> {
         resolver 8.8.8.8;
         set $backend "<%= route %>";
         proxy_pass $backend;
+        proxy_set_header Cache-Control "public, maxage=600";
```
Comment (ccostino) [inline nginx.conf]: I was just double checking this, it looks like we want `max-age` according to the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)

**Pull request merged**